indico/modules/events/client/js/reviewing/components/TimelineHeader.jsx

```diff
@@ -69,4 +89,5 @@ TimelineHeader.propTypes = {
   }).isRequired,
   eventId: PropTypes.number.isRequired,
   children: PropTypes.node.isRequired,
+  reviewConditionsOk: PropTypes.bool.isRequired,
```

Comment:
This prop is marked as required but not defined in `events/papers/clients/js/components/Paper.jsx`.

Reply:
Thanks @plourenco for pointing out, I will keep a default value for this Prop.